backend/app/adapter/sqldb/short_link_integration_test.go

```diff
@@ -326,11 +326,14 @@ func TestShortLinkSql_UpdateShortLink(t *testing.T) {
 						testCase.oldAlias,
 						testCase.newShortLink,
 					)
+					assert.Equal(t, nil, err)
 
+					shortLink, err = shortLinkRepo.GetShortLinkByAlias(testCase.newShortLink.Alias)
 					if testCase.hasErr {
 						assert.NotEqual(t, nil, err)
 						return
 					}
+
```

Comment:
```suggestion
```

Reply:
It's good to have clear logic separation through blank lines.